build/int.cloudbuild.yaml

```diff
@@ -43,6 +43,37 @@ steps:
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSapHanaHaSimpleModule --stage teardown --verbose']
 
+- id: apply sap-hana-simple
+  waitFor: ["init-all", "teardown sap-hana-ha-simple"]
```

Comment:
Is this expected to run serially since we are also waiting for `teardown sap-hana-ha-simple`?

Reply:
Yes because we're running into quota limits. SAP requires sizable machines and the default 72 CPU quota is not enough. To get around that we're running them serially.

Reply:
Gotcha, in that case (just for maintainability) we can actually remove all waitFor and it will run through steps in order.